src/QMCWaveFunctions/Fermion/DiracDeterminantCUDA.h

```diff
@@ -149,22 +149,32 @@ class DiracDeterminantCUDA : public DiracDeterminantBase
 
 public:
   // safe-guard all CPU interfaces
-  DiracDeterminantCUDA* makeCopy(SPOSet* spo) const { APP_ABORT("Calling DiracDeterminantCUDA::makeCopy is illegal!"); }
+  DiracDeterminantCUDA* makeCopy(SPOSet* spo) const
+  {
+    APP_ABORT("Calling DiracDeterminantCUDA::makeCopy is illegal!");
```

Comment:
As per #125 I think this should be a throw. As well as the following APP_ABORTS.

Reply:
Changed.